ddl/table.go

```diff
@@ -72,6 +72,42 @@ func onCreateTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error)
 	}
 }
 
+func onCreateView(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error) {
+	schemaID := job.SchemaID
+	tbInfo := &model.TableInfo{}
+	var orReplace bool
+	if err := job.DecodeArgs(tbInfo, &orReplace); err != nil {
+		// Invalid arguments, cancel this job.
+		job.State = model.JobStateCancelled
+		return ver, errors.Trace(err)
+	}
+	tbInfo.State = model.StateNone
+	err := checkTableNotExists(t, job, schemaID, tbInfo.Name.L)
+	if err != nil && !orReplace {
```

Comment:
The process of `orReplace` in this PR is not correct although all the phenomenon of `CreateView orReplace` and `Select from View` may look well. We'd better leave the implementation of `OrReplace` in an individual PR.